Figures/Helper.cs

```diff
@@ -0,0 +1,271 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualBasic.CompilerServices;
+
+namespace FiguresProject
+{
+    internal class Helper
+    {
+        public static void PrintMenu()
+        {
+            Console.WriteLine("1. Add new figure");
+            Console.WriteLine("2. Change Figure parameters");
+            Console.WriteLine("3. Remove figure");
+            Console.WriteLine("4. Draw the figure");
+            Console.WriteLine("5. Exit");
+        }
+
+        public static void AddFigure(List<Figures> figureList)
+        {
+            string[] userChoice = CollectNewFigureData();
+            int figureType = Convert.ToInt32(userChoice[0]);
+            int isCustom = Convert.ToInt32(userChoice[1]);
+            const int isFillParam = 2, figurePointParam = 3;
+            switch (figureType)
+            {
```

Comment:
Read about upcasting and downcasting. can be very useful here as adding figures to the list should be as Figure.
A wider explanation could be provided after said research.